src/config/config.js

```diff
@@ -9,11 +9,5 @@ module.exports = {
     port: process.env.PGPORT,
     host: '127.0.0.1',
     dialect: 'postgres',
-    dialectOptions: {
```

Comment:
You really want to change this at the same time as the PR, and in production?

Reply:
This if for development, not prod?

Reply:
The change shouldn't affect prod